app/helpers/application_helper.rb

```diff
@@ -16,7 +16,7 @@ def swap_icon_url
   def pretty_time(time, format: nil)
     return unless time
     content_tag(:time, datetime: time.utc.iso8601, title: time.utc.strftime("%Y-%m-%d %H:%M %Z")) do
-      time.strftime(format || current_user.try(:time_display) || TIME_FORMAT)
+      time.in_time_zone.strftime(format || current_user.try(:time_display) || TIME_FORMAT)
```

Comment:
do. are we not timezoning that properly. oh no.

Reply:
weirdly it seems like it's still working some places...?

> A fix to the default format for pretty_time (it previously didn't always use in_time_zone and at some point boards#index has regressed to using the wrong timezone for its update time? post lists seem fine still...)